ocs_ci/templates/workloads/cephfs_stress/cephfs_filesystem_monitor.py

```
@@ -0,0 +1,633 @@
+        logger.info(f"[{monitor_name}] Monitoring {len(sample_dirs)} sampled directories")
+        
+        # Monitor each sampled directory
+        all_success = True
+        for sample_dir in sample_dirs:
+            if shutdown_event.is_set():
+                break
+                
+            logger.info(f"[{monitor_name}] Checking: {sample_dir}")
+            
+            # ls -lRf on sample directory
+            success, output, duration = run_command_with_timeout(
```

Ruff (0.15.12)

[warning] Unpacked variable `output` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)